src/compat/attribute.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_COMPAT_ATTRIBUTE_H
+#define BITCOIN_COMPAT_ATTRIBUTE_H
+
+#if defined(__GNUC__)
+#define FORCED_INLINE inline __attribute__((always_inline))
```

Comment:
If we are going to add this, `ALWAYS_INLINE` seems like a better name.

Reply:
Thanks!. [Updated](https://github.com/bitcoin/bitcoin/pull/24773#issuecomment-1329464328).